fe1-web/src/features/social/context/SocialMediaContext.ts

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import { PublicKey } from 'core/objects';
+
+/**
+ * The context that allows access to the current user public key everywhere in the
+ * feature.
+ */
+export const SocialMediaContext = React.createContext({
+  currentUserPublicKey: new PublicKey(''),
```

Comment:
I don't think so, because it gets updated as soon as we change lao. You can see that in [SocialMediaNavigation](https://github.com/dedis/popstellar/blob/work-fe1-kschneiter-inject-dependencies-social/fe1-web/src/features/social/navigation/SocialMediaNavigation.tsx), lines 48 to 76.